Makefile

```
@@ -535,9 +535,10 @@ run-builder: ## Drop into a shell inside a container running the cilium-builder
 
 .PHONY: renovate-local
 renovate-local: ## Run a local linter for the renovate configuration
-	$(CONTAINER_ENGINE) run --rm -ti \
+	@echo "Running renovate --platform=local"
+	@$(CONTAINER_ENGINE) run --rm -ti \
```

nit: Verbosity variable V=0 vs V=1 controls $QUIET that then allows either silence or echoing the full command depending on how you invoke the make command. 

```suggestion
	$(QUIET)$(CONTAINER_ENGINE) run --rm -ti \
```

---

I wanted to avoid printing the user's token by default,  it seems like `$(QUIET)` defaults to printing the output though.

---

Ah! Yeah fair trade-off.

---

I figured out how to set `V=0` by default for this target specifically, with the option to override.